features/view_specs/view_spec.feature

```diff
@@ -212,3 +212,24 @@ Feature: view spec
       """
     When I run `rspec spec/views`
     Then the examples should all pass
+
+  Scenario: path helpers should take precedence over asset helper
```

Comment:
Take a look at how we check `params` / `template` etc is tested, but if you absolutely need to run a spec to test this then you need to check the `run_count` and check for failures, you can't rely on output it'll be swallowed by the null object.